Tasks/ServiceFabricPowerShellV1/ServiceFabricPowerShell.ps1

```diff
@@ -32,7 +32,7 @@ $certificate = $null
 try
 {
     # Initialize Service Fabric.
-    Import-Module $PSScriptRoot\ps_modules\ServiceFabricHelpers
+    Import-Module $PSScriptRoot\ps_modules\ServiceFabricHelpers -DisableNameChecking
```

Comment:
Along with this, should we also fix the names in ServiceFabricHelpers

Reply:
done